ownCloud/Resources/Info.plist

```diff
@@ -29,8 +33,14 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
+	<key>NSAppleMusicUsageDescription</key>
+	<string>We need this so that you can upload photos and videos from your photo library</string>
 	<key>NSFaceIDUsageDescription</key>
 	<string>Unlock the app without entering the passcode.</string>
+	<key>NSPhotoLibraryAddUsageDescription</key>
+	<string>We need this so that you can upload photos and videos from your photo library</string>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>We need this so that you can upload photos and videos from your photo library</string>
```

Comment:
Alternative suggestion:
This permission is needed by the ownCloud app to upload photos and videos from your photo library.

Reply:
Done.